### DISCUSSION TITLE
No pairing again after samsung system update 010924

### DISCUSSION THREAD

**Discussion by oracess** [NightscoutFoundation/xDrip#3698] [Bug reports]
No pairing again after samsung system update 2024.09.27 like #2902 after 8 months working well. Pairing with Dexcom-App possible and using latest xdrip+ from 2024.09.27!

**Comment by Navid200** [comment]
I use a Pixel 6a with G7.

After it pairs with a new G7, I will have to restart the phone.  Otherwise, it may go into a continuous disconnect mode where it will give no readings and show errors on the status page.

Is it that, or does it really not pair at all?

I have also seen people claim that enabling trust auto connect has helped with some Samsungs.
